src/matrixrtc/MembershipManager.ts

```diff
@@ -18,13 +18,30 @@ import { MembershipConfig } from "./MatrixRTCSession.ts";
  * @internal
  */
 export interface IMembershipManager {
+    /**
+     * Returns true if we intend to be participating in the MatrixRTC session.
+     * This is determined by checking if the relativeExpiry has been set.
```

Comment:
```suggestion
```

Because *how* it is implemented is up to the `class` that implements the `interface`.

Reply:
true...